.github/workflows/azure-static-web-apps-lively-hill-0d219190f.yml

```diff
@@ -3,16 +3,23 @@ name: Azure Static Web Apps CI/CD
 on:
   push:
     branches:
-      - main
+      - test
   pull_request:
     types: [opened, synchronize, reopened, closed]
     branches:
-      - main
+      - test

```

Comment:
Let's go ahead and remove the pull request trigger for now. We'll circle back and look at adding this later.